org.KangLin.Rabbit.RemoteControl.json

```diff
@@ -0,0 +1,262 @@
+/* See: https://docs.flatpak.org/en/latest/manifests.html */
+/* See: https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html#flatpak-manifest */
+{
+    "id": "org.KangLin.Rabbit.RemoteControl",
```

Comment:
`kanglin.org` doesn't exist. How can you claim it's under the control of the developers?